src/applications/vaos/appointment-list/components/FutureAppointmentsList.jsx

```diff
@@ -112,15 +110,8 @@ function FutureAppointmentsList({
     );
   }
 
-  const header = !expressCare.hasRequests && (
```

Comment:
These headers are below the tab, which I don't think is what we expect, currently.